frontend/src/metabase/components/MetadataInfo/ColumnInfoIcon/ColumnInfoIcon.tsx

```diff
@@ -10,19 +10,28 @@ import {
   QueryColumnInfoPopover,
   TableColumnInfoPopover,
 } from "../ColumnInfoPopover";
-import { PopoverHoverTarget, HoverParent } from "../InfoIcon";
+import { PopoverHoverTarget, HoverParent, defaultDelay } from "../InfoIcon";
 export { HoverParent };
 
+export type QueryColumnInfoIconProps = QueryColumnInfoPopoverProps & {
+  showIfEmpty?: boolean;
```

Comment:
This prop is no longer used in app code (only in tests), we can remove it (and the tests that use it).

Same goes for `TableColumnInfoIconProps`